boxoffice/forms/order.py

```diff
@@ -35,3 +35,13 @@ class BuyerForm(forms.Form):
     email = forms.EmailField(__("Email"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
     fullname = forms.StringField(__("Full name"), validators=[forms.validators.DataRequired()])
     phone = forms.StringField(__("Phone number"), validators=[forms.validators.Length(max=16)])
+
+
+class OrderSessionForm(forms.Form):
+    utm_campaign = forms.StringField(__("UTM Campaign"))
+    utm_source = forms.StringField(__("UTM Source"))
+    utm_medium = forms.StringField(__("UTM Medium"))
+    utm_term = forms.StringField(__("UTM Term"))
+    utm_content = forms.StringField(__("UTM Content"))
+    utm_id = forms.StringField(__("UTM Id"))
+    gclid = forms.StringField(__("Gclid"))
```

Comment:
> You can't fail validation as there's no user input.

@jace I'm a little confused because I was under the impression that the validation was supposed to be pushed down to the database, because of the above comment.